src/config.ts

```diff
@@ -100,6 +100,10 @@ export const LIQUIDITY_POOL_REGISTRY_ADDRESS: string | undefined = _.isEmpty(
           EnvVarType.ETHAddressHex,
       );
 
+export const RFQT_API_KEY_WHITELIST: string[] = (process.env.RFQT_API_KEY_WHITELIST || '[]').split(',');
```

Comment:
If `process.env.RFQT_API_KEY_WHITELIST` is `undefined` then `RFQT_API_KEY_WHITELIST` will be `[ '[]' ]` when it should really just be an empty array.  Seems like a relic from the old JSON approach

```
> const RFQT_API_KEY_WHITELIST: string[] = (process.env.RFQT_API_KEY_WHITELIST || '[]').split(',');
undefined
> RFQT_API_KEY_WHITELIST
[ '[]' ]
```

Reply:
Thank you for catching this.  Addressed in https://github.com/0xProject/0x-api/pull/162/commits/e1693bba254b24852fbedae88b25bf8ebedc5616